resources/js/features/user-game-activity/components/UserGameActivityTimeline/ClientLabel/ClientLabel.tsx

```diff
@@ -0,0 +1,59 @@
+import type { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+import { LuMonitor } from 'react-icons/lu';
+
+import { cn } from '@/common/utils/cn';
+import { buildEmulatorClientLabel } from '@/features/user-game-activity/utils/buildEmulatorClientLabel';
+
+import { ActivityBasicLabel } from '../../ActivityBasicLabel';
+import { ActivityTooltipWrapper } from '../../ActivityTooltipWrapper';
+import { ManualUnlockLabel } from './ManualUnlockLabel';
+
+interface ClientLabelProps {
+  session: App.Platform.Data.PlayerGameActivitySession;
+}
+
+export const ClientLabel: FC<ClientLabelProps> = ({ session }) => {
+  const { t } = useTranslation();
+
+  if (session.type === 'reconstructed') {
+    return (
+      <ActivityTooltipWrapper
+        Icon={LuMonitor}
```

Comment:
Should this be off-colored too, to distinguish it from the actual rich presence?

Reply:
RATools uses a [map marker](https://github.com/Jamiras/RATools/blob/master/Source/Resources/rich_presence.png) for Rich Presence. 

Could also use a paper map, or pulse wave, or leave it as-is. I don't really have a strong opinion.

Reply:
I think `LuMonitor` should remain the time to be consistent with the other labels in the heading:
![Screenshot 2025-01-01 at 3 01 50 PM](https://github.com/user-attachments/assets/edb98359-aee8-46d5-a168-757c2312edb3)

For the rich presence icon, I also think it may be best to stay as-is, but this isn't a terribly strong opinion. To me, the current icon conveys that something was "broadcast" from the session:
![Screenshot 2025-01-01 at 3 02 40 PM](https://github.com/user-attachments/assets/f50ba253-3c34-4585-88b6-25c008616470)